tests/test_l10n_download_links.py

```diff
@@ -6,8 +6,6 @@
 import pytest
 import requests
 
-pytestmark = [pytest.mark.nondestructive, pytest.mark.skip_selenium]
-
 
 def pytest_generate_tests(metafunc):
     base_url = metafunc.config.option.base_url
```

Comment:
This is causing an error for me. I am not passing `--base-url` on the command line, but rather am allowing it to be picked up from `setup.cfg`. I don't think this code here will find it when that is the source, so I end up with `base_url == None` in these tests, which then throws an error:

`MissingSchema: Invalid URL 'None/firefox/all/': No schema supplied. Perhaps you meant http://None/firefox/all/?`

This is not necessarily a problem with this file, as we will always pass a `--base-url` when running the tests on Jenkins, but I wanted to point it out. I can cause failures locally if you run like I do and just allow the default `setup.cfg` to be used.